kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/scheduler/KafkaScheduler.java

```diff
@@ -180,40 +178,37 @@ private void startApiServer() {
             }
         });
     }
+    // dcos-commons 0.8.1 upgrade: just to be consistent with the existing java tests (see KafkaSchedulerTest.java)
```

Comment:
Comment shouldn't be needed, the `protected` should be enough to imply that we expect the return value to be overridden